src/main/treeDataProviders/dependenciesTree/dependenciesTreeFactory.ts

```diff
@@ -25,6 +26,8 @@ export class DependenciesTreesFactory {
             workspaceFolders,
             treesManager.logManager
         );
+
+        await this.sendUsageReport(projectDescriptors, treesManager.connectionManager);
```

Comment:
I think to maximize the performance we should send usage report asynchronously, as we do in the JFrog CLI.
```suggestion
        this.sendUsageReport(projectDescriptors, treesManager.connectionManager);
```